app/models/team.rb

```diff
@@ -0,0 +1,4 @@
+class Team < ApplicationRecord
+  validates :name, length: { minimum: 4, maximum: 50 }
+  validates :round_duration, numericality: { only_integer: true, greater_than: 0 }
```

Comment:
What's the point of adding the "numericality"/only integer constraint to an integer attribute? Are we just showing off newly learned skills? 😄👨‍💻